### PR TITLE
DL-3270 fix accessibility validator issues

### PR DESCRIPTION
**Pull request by andy-p79**
# DL-2977 - > DL-3270 fix accessibility issues identified from validator tool

**Bug fix ** 
Fixes in response to accessibility issues identified by the validator
Fixed h2/h3 issue
Fixed issue with duplicate IDs
Fixed issue with some  divs wrapping fieldset content
Removed aria-role radiogroup from fieldset.
Corrected aria-described by on additional address form

## Checklist

Reviewee (Andy)

 - [x]  I've made every effort to commit high quality, clean code and I have executed relevant static analyses to be sure
 - [x]  I've executed the acceptance test pack locally to ensure there are no functional regressions
 - [x]  I've added my code using logical, atomic commits, squashing as appropriate - including the JIRA issue number in the commit message

### CODE REVIEW

File: app/views/helpers/awrsInputTypeGroupCheckbox.scala.html [hunk 1]
```diff
@@ -86,27 +86,25 @@
...
-                                aria-describedby="@{params.ariaDescribedBy}"
-                           } else {
-                                aria-describedby="@(params.field.id)-legend"
-                           }
-                           @inputName
-                           value="@checkboxOptions._1" @if(values.contains(checkboxOptions._1)){checked="checked"}>
-                    @checkboxOptions._2
-                </label>
-            }
+<fieldset @fieldSetClass @fieldSetAttributes role="group" @if(fieldErrors.nonEmpty) {aria-labelledby="@params.field.name-error-0"}>
```
Comment (paulanderson1234) [inline app/views/helpers/awrsInputTypeGroupCheckbox.scala.html:89]: I think  role="group" this needs removing if possible as inferred by VNU issues on report

File: app/views/includes/awrs_main.scala.html [hunk 2]
```diff
@@ -56,7 +56,7 @@
     <script type="text/javascript" src='@routes.Assets.at("javascripts/awrs.js")'></script>
```
Comment (paulanderson1234) [inline app/views/includes/awrs_main.scala.html:56]: I believe we need to remove all instances of type="text/javascript" based on VNU reported incidents where possible

**Comment by paulanderson1234** [comment]
Need to look at the following:
 <div> <legend id="thirdPartyStorage-legend"> on trading-activity as apparently not allowed as child element of <div>

the use of autocomplete="off" for certain input types and type"radio" has been identified in awrsInputTypeRadioGroup (line 152) so we should probably try and remove this

File: app/views/helpers/awrsInputTypeGroupCheckbox.scala.html (hunk 1 above)
Comment (andy-p79) [reply app/views/helpers/awrsInputTypeGroupCheckbox.scala.html:89]: I'd removed the role completely - on a fieldset the aria role is already implicitly set as group so it doesn't need to be assigned

File: app/views/includes/awrs_main.scala.html (hunk 2 above)
Comment (andy-p79) [reply app/views/includes/awrs_main.scala.html:56]: In the report I was looking at the error that refers to the charset
VNU:Element “script” must not have attribute “charset” unless attribute “src” is also specified. (which i fixed by removing that from the inline script block)

In HTML5 text/javascript is the default type anyway, so I can take out if need - however there is a chance that older browsers that don't support HTML5 might have issue with this removed (Required in HTML 4 Spec) Would be interested to see if this is flagged again

**Comment by andy-p79** [comment]
Let me check the code around the div legend id="thirdPartyStorage-legend"
I'd removed the div wrappers from these fieldsets in both the awrsInputTypeGroupCheckbox.scala.html and the awrsInputTypeRadioGroup.scala.html.

The autocomplete  - I'd fixed the VNU: Attribute “autocomplete” not allowed on element “label” at this point, but wasn't seeing the issue in local dev for the input.

Will look at the input field autocomplete for those types but I'm be surprised if that is an issue  - Unlike other browsers, Firefox by default persists the dynamic checked state of an <input> across page loads. The autocomplete attribute can be used to control this feature.
(https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/radio#Additional_attributes)

File: app/views/includes/awrs_main.scala.html (hunk 2 above)
Comment (paulanderson1234) [reply app/views/includes/awrs_main.scala.html]: All removed

**Pull request merged**